lib/rubycritic/source_control_systems/git.rb

```diff
@@ -21,7 +21,7 @@ def git(arg)
       end
 
       def self.supported?
-        git('branch 2>&1') && $?.success?
+        git('branch 2>&1') && $CHILD_STATUS.success?
```

Comment:
This is breaking the tests. We must `require "English"` so that this works  (see https://github.com/rubocop-hq/rubocop/issues/1747).

You can add the require to the top level rubycritic module I think.

Reply:
Fixing this.

> You can add the require to the top level rubycritic module
I was going to require only in the files that used that variable. Would requiring in the top module be better for some reason?

Reply:
Just because it might be used elsewhere. But I guess if this is the only file that uses `$CHILD_STATUS` then we can leave the require here.

Reply:
I mean: if `$CHILD_STATUS` is only used on `RubyCritic::SourceControlSystem` then it can be imported required only in this module.